components/proxy/src/main/java/com/hotels/styx/metrics/reporting/graphite/GraphiteConfig.java

```diff
@@ -30,16 +30,19 @@
     private final int port;
     private final long intervalMillis;
     private final String prefix;
+    private final boolean enabled;
 
     @JsonCreator
     GraphiteConfig(@JsonProperty("host") String host,
                    @JsonProperty("port") Integer port,
                    @JsonProperty("intervalMillis") Long intervalMillis,
-                   @JsonProperty("prefix") String prefix) {
+                   @JsonProperty("prefix") String prefix,
+                   @JsonProperty("enabled") String enabled) {
```

Comment:
This should probably be a `boolean` or a `Boolean` to be null safe. Either way it should be the same type as the getter.

Reply:
What led me to use a `String` is to default the value to `true` when the `enabled` property is missing (to maintain backward compatibility) while using a `boolean` will always default to `false` with `@JsonCreator` if the property is missing.